src/mtcp/mtcp_restart.c

```diff
@@ -1305,7 +1305,8 @@ skip_lh_memory_region_ckpting(const Area *area, LowerHalfInfo_t *lh_info)
       mtcp_strstr(area->name, "/dev/xpmem") ||
       mtcp_strstr(area->name, "/dev/shm") ||
       mtcp_strstr(area->name, "/SYS") ||
-      area->addr == lh_info->startData) {
+      area->addr == lh_info->startData ||
+      ((lh_info->startBSS != NULL) && area->addr == lh_info->startBSS)) {
```

Comment:
Maybe move this condition to a const bool above line 1302.

```C
const bool is_lh_bss_region = (lh_info->startBSS != NULL) && area->addr == lh_info->startBSS);
```

Then, you can reference the bool variable here. It'll improve code readability.